test/config/ApplicationConfigSpec.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import test_utils.TAMCSetupSpec
+
+class ApplicationConfigSpec extends TAMCSetupSpec {
+
+  "check rates for earliest valid year" when {
+
+    "return valid year" in {
+      ApplicationConfig.TAMC_BEGINNING_YEAR should be(2016)
+    }
+
+  }
+
```

Comment:
We should also add the test for the actualTaxYear method that is being used as this has been changed slightly. At the least!

Reply:
I didn't change the logic for actualTaxYear I just reformat it